pkg/core/auth/authz/policy/verified.go

```diff
@@ -0,0 +1,23 @@
+package policy
+
+import (
+	"net/http"
+
+	"github.com/skygeario/skygear-server/pkg/core/auth"
+	"github.com/skygeario/skygear-server/pkg/core/auth/authz"
+)
+
+// DenyNotVerifiedUser denies not verified user.
+func DenyNotVerifiedUser(r *http.Request, ctx auth.ContextGetter) error {
+	authInfo, _ := ctx.AuthInfo()
+
+	if authInfo != nil && authInfo.Verified {
```

Comment:
We should follow the convention of DenyDisabledUser. If authInfo is nil, return nil